### PR TITLE
[Lang] Fix pylint rule W0401

### PR DESCRIPTION
**Pull request by jyz0309**
Related issue #3151

### CODE REVIEW

File: python/taichi/lang/ir_builder.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import astor
 from taichi.lang import impl
 from taichi.lang.ast.symbol_resolver import ASTResolver
-from taichi.lang.ast_builder_utils import *
+from taichi.lang.ast_builder_utils import LoopStatus
```
Comment (sjwsl) [inline python/taichi/lang/ir_builder.py:9]:
```suggestion
from taichi.lang.ast_builder_utils import Builder, LoopStatus
```

**Comment by sjwsl** [comment]
Thanks for your help! Could you update your PR title by https://docs.taichi.graphics/lang/articles/contribution/contributor_guide#pr-title-format-and-tags?

**Review by sjwsl — APPROVED** [review]
LGTM. Thanks!

**Comment by jyz0309** [comment]
@ailzhang CI pass. Can you help me to merge?

**Pull request merged**